### PR TITLE
Expose ConfiguredSystems

### PR DESCRIPTION
**Pull request by bdunne**
DEPENDS ON: https://github.com/ManageIQ/manageiq/pull/19928

### CODE REVIEW

File: spec/requests/providers_spec.rb [hunk 1]
```diff
@@ -1837,6 +1837,52 @@ class DummyProvider; end
...
+
+        get(api_provider_configured_systems_url(nil, ems))
+
+        expect(response).to have_http_status(:forbidden)
+      end
+    end
+
+    context 'GET /api/providers/:id/configured_systems/:s_id' do
+      it 'returns the configured_systems with an appropriate role' do
+        api_basic_authorize action_identifier(:providers, :read, :resource_actions, :get)
```
Comment (abellotti) [inline spec/requests/providers_spec.rb]: Lookin' at configuration profiles, I `think` this should be action_identifier(:configured_systems, :read, :subresource_actions, :get)

**Comment by abellotti** [comment]
LGTM @bdunne 👍  Thanks for the API Enhancement !!

**Comment by abellotti** [comment]
Yeay, happy 🟢 travis.

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit 057d657472609d9a83210497d928f30aa54b3260
Author: Alberto Bellotti <abellotti@users.noreply.github.com>
Date:   Wed Mar 4 16:12:12 2020 -0500

    Merge pull request #765 from bdunne/expose_configured_systems

    Expose ConfiguredSystems

    (cherry picked from commit d304a6f2e129a960f8852484308716ed509af7ae)
```

**Pull request merged**